BrowserKit/Sources/SiteImageView/ImageProcessing/BundleImageFetcher/BundleImageFetcher.swift

```diff
@@ -83,9 +83,12 @@ class DefaultBundleImageFetcher: BundleImageFetcher {
          into the BrowserKit package, this would have resulted into macking a lot of changes into accessors to be able to make our own file outside of the BrowserKit context.
          */
 
-        let schemeStrippedFinancialUrlString = Environment.current.urlProvider.financialReports.absoluteString.replacingOccurrences(of: "https://", with: "")
-        if domain.bundleDomains.contains(schemeStrippedFinancialUrlString) {
-            return "blog.ecosia.finance"
+        let financialReportsURL = Environment.current.urlProvider.financialReports.absoluteString.replacingOccurrences(of: "https://", with: "")
+        let privacyURL = Environment.current.urlProvider.privacy.absoluteString.replacingOccurrences(of: "https://", with: "")
+        let foo = [financialReportsURL : "blog.ecosia.finance",
```

Comment:
Did you mean to leave `foo` here? Would personally prefer something more descriptive like `urlMap`.

Reply:
@lucaschifino deinitely slipped! Renaming it at lightning speed!